Resources/Private/Templates/VarnishCache/SearchForNode.html

```diff
@@ -27,24 +27,25 @@
 									</td>
 								</tr>
 							</f:if>
-							<f:for each="{site.nodes}" as="node">
-								<f:alias map="{url: '{neos:uri.node(node: node, absolute: 1)}'}">
+							<f:for each="{site.nodes}" as="item">
+								<f:alias map="{url: '{neos:uri.node(node: item.node, absolute: 1)}'}">
 									<tr class="fold-{site.site.nodeName}">
-										<td title="{node.identifier}">
-											<f:if condition="{node.nodeType.fullConfiguration.ui.icon}">
-												<i title="{node.nodeType.label}" class="{node.nodeType.fullConfiguration.ui.icon}"></i>
+										<td title="{item.node.aggregateId}">
+
+											<f:if condition="{item.nodeTypeIcon}">
+												<i title="{item.nodeTypeLabel}" class="{item.nodeTypeIcon}"></i>
 											</f:if>
-											<a href="{url}" target="_blank">{node.label}</a>
+											<a href="{url}" target="_blank">{neos:node.label(node: item.node)}</a>
 										</td>
-										<td title="{node.path}"><a href="{url}" target="_blank">{neos:uri.node(node: node)}</a></td>
-										<td><f:if condition="{node.properties.disableVarnishCache}">
+										<td title="{item.node.path}">2<a href="{url}" target="_blank">{neos:uri.node(node: item.node)}</a></td>
+										<td><f:if condition="{item.node.properties.disableVarnishCache}">
 											<f:then><i class="fa fa-warning-sign"></i> Caching disabled</f:then>
-											<f:else>{f:if(condition: node.properties.cacheTimeToLive, then: '<span title="Time-to-live"><i class="fa fa-time"></i> {node.properties.cacheTimeToLive} s')}</span></f:else>
+											<f:else>{f:if(condition: item.node.properties.cacheTimeToLive, then: '<span title="Time-to-live"><i class="fa fa-time"></i> {item.node.properties.cacheTimeToLive} s')}</span></f:else>
 										</f:if></td>
 										<td class="neos-action">
-											<button class="neos-button" data-toggle="modal" href="#{node.identifier}" title="Additional info" data-check-url="{f:uri.action(action: 'checkUrl', format: 'json', arguments: {url: url})}"><i class="fa fa-info-circle"></i></button>
-											<f:link.action action="purgeCache" format="json" arguments="{node: node}" class="neos-button clear-cache" title="Clear cache" data="{node-label: node.label}"><i class="fa fa-bolt"></i></f:link.action>
-											<div class="neos-hide" id="{node.identifier}">
+											<button class="neos-button" data-toggle="modal" href="#{item.node.aggregateId}" title="Additional info" data-check-url="{f:uri.action(action: 'checkUrl', format: 'json', arguments: {url: url})}"><i class="fa fa-info-circle"></i></button>
+											<f:link.action action="purgeCache" format="json" arguments="{nodeAddress: item.nodeAddress}" class="neos-button clear-cache" title="Clear cache" data="{node-label: item.node.label}"><i class="fa fa-bolt"></i></f:link.action>
```

Comment:
fyi this link generation here in combination with

```
NodeAddress::fromNode($node)->toJson()
```

in the varnishCacheController caused some discussion in https://neos-project.slack.com/archives/C04PYL8H3/p1736364381060509

Ill take care of figuring out what we want for the Neos 9 release - either having to use nodeAddress here manually or reallowing some autocasting magic in the back and passing a full node -> https://github.com/neos/neos-development-collection/issues/5069

But for now, this code works and is used in other places as well. Its not wrong its just less magic and more explicit^^